nlpretext/textloader/textloader.py

```diff
@@ -24,11 +43,20 @@ def __init__(
             encoding of the text to be loaded, can be utf-8 or latin-1 for example
         file_format: string | None
             format of the files to be loaded
+        use_dask: bool
+            use dask to load text
         """
         self.text_column = text_column
         self.encoding = encoding
         self.file_format = file_format
 
+        if 'dask' in sys.modules and use_dask:
+            self.loader = daskloader
+            self.use_dask = True
+        else:
```

Comment:
Can you add an warning message if "use_dask=True" is specified but dask is not installed, to warn the user that pandas will be used anyway ?

Reply:
yep very good point